test/WebSites/InlineConstraintsWebSite/Startup.cs

```diff
@@ -12,6 +15,27 @@ public class Startup
         // Set up application services
         public void ConfigureServices(IServiceCollection services)
         {
+
+            services.Configure<RouteOptions>(
+                routeOptions => routeOptions.ConstraintMap.Add(
+                    "producttype",
+                    typeof(ItemTypeConstraint)));
+
+            services.Configure<RouteOptions>(
+                routeOptions => routeOptions.ConstraintMap.Add(
+                    "servicetype",
+                    typeof(ItemTypeConstraint)));
+
+            services.Configure<RouteOptions>(
+                routeOptions =>
+                {
+                    if (routeOptions.ConstraintMap.ContainsKey("servicetype"))
+                    {
+                        routeOptions.ConstraintMap["servicetype"] =
+                            typeof(ItemTypeUpperCaseConstraint);
+                    }
+                });
```

Comment:
I think updating an existing constraint is not a good idea. this same WebSite is used in many existing tests, and I don't like to disturb them. I'd like to keep this for that reason.